source/blog/2017-08-16-ovirt-ansible-roles-how-to-use.html.md

```diff
@@ -0,0 +1,149 @@
+
+```bash
+$ sed -i 's|#roles_path    = /etc/ansible/roles|roles_path = /etc/ansible/roles:/etc/ansible/roles/ovirt.ovirt-ansible-roles/roles|'  /etc/ansible/ansible.cfg
+```
+
+Now copy one of the examples from directory `/etc/ansible/roles/ovirt.ovirt-ansible-roles/examples/` into your working directory modify the needed variables and run the playbook.
+
+
+## oVirt Ansible Roles RPM
+
+In the latest oVirt repositories you can find `ovirt-ansible-roles` package. In order to have oVirt repositories available you need to install `ovirt-release` RPM:
+
+```bash
+$ yum install -y http://resources.ovirt.org/pub/yum-repo/ovirt-release41.rpm
+```
+
+oVirt release repository now provides `ovirt-ansible-roles` package:
+
+```bash
+$ yum install -y ovirt-ansible-roles
+```
+
+The structure of `ovirt-ansible-roles` package is following:
+
+ - __/usr/share/ansible/roles__ - stores the roles
+ - __/usr/share/doc/ovirt-ansible-roles/__ - stores the examples, basic overview and licence
+ - __/usr/share/doc/ansible/roles/{role_name}__ - store the documentation specific for role
+
+Now copy one of the examples from directory `/usr/share/doc/ovirt-ansible-roles/examples/` into your working directory modify the needed variables and run the playbook.
+
+## Example of tweaking and running the playbook
+
+## Create example playbook
+
+The proccess of runnning the example from galaxy and from RPM is bit different so please follow relevant part.
+
+### Ansible Galaxy
+
+I suppose you followed the part how to install the roles from galaxy, so you roles are already installed in `/etc/ansible/roles/ovirt.ovirt-ansible-roles/roles/` and you have tweaked the `ansible.cfg` to look for those roles.
+
+Now we will run a role called `ovirt-image-template` to download the qcow2 image and create a template from it.
+
+Copy the `ovirt-image-template` role example, I will use `/tmp` as working directory for this example:
+
+```bash
+$ cd /tmp
+$ cp /etc/ansible/roles/ovirt.ovirt-ansible-roles/examples/ovirt_image_template.yml .
+```
+
+### oVirt Ansible Roles RPM
+I suppose you followed the part how to install the roles from RPM, so you roles are already installed in `/usr/share/ansible/roles`.
+
+Now we will run a role called `ovirt-image-template` to download the qcow2 image and create a template from it.
+
+Copy the `ovirt-image-template` role example, I will use `/tmp` as working directory for this example:
```

Comment:
Suggest:
"Copy the `ovirt-image-template` role example. In this example I will use `/tmp` as the working directory:"